index.d.ts

```diff
@@ -1,3 +1,12 @@
+export interface Options {
+	/**
+	Count [Ambiguous characters](https://www.unicode.org/reports/tr11/#Ambiguous) as a width of 1, false counts as 2.
```

Comment:
```suggestion
	Count [ambiguous width characters](https://www.unicode.org/reports/tr11/#Ambiguous) as having narrow width (count of 1) instead of wide (count of 2).
```